R/formula-ac.R

```diff
@@ -580,10 +587,21 @@ use_ac_cov_time <- function(x) {
 
 # does the model need latent residuals for autocor structures?
 has_ac_latent_residuals <- function(bterms) {
-  !has_natural_residuals(bterms) &&
+  (!has_natural_residuals(bterms) ||
+     has_ac_subset(bterms, latent = T)) &&
     (use_ac_cov(bterms) || has_ac_class(bterms, "arma"))
 }
 
+# use explicitly parameterized autocor effects?
```

Comment:
remember to remove this before we merge if we continue to not need this function.